ods_ci/tasks/Tasks/rhods_olm.robot

```diff
@@ -23,6 +23,7 @@ Can Install RHODS Operator
   [Tags]  install
   IF  "${PRODUCT}" == "ODH"
       Set Global Variable  ${OPERATOR_NAMESPACE}  opendatahub-operators
+      Set Global Variable  ${OPERATOR_NAME_LABEL}  opendatahub-operator
```

Comment:
What about the else branch? Or is this variable defined with some default value somewhere else already?

Reply:
thanks @jstourac I had the else part but I didnt commited. Pushing now